.github/codeowners

```
@@ -1 +1 @@
-* @HarryET
\ No newline at end of file
+* @WalletConnect/rust-team
```

I think we should make this more limited as to not ping everyone on the team who doesn't have ability to understand what they are approving.

```suggestion
* @geekbrother
* @chris13524
```

---

Makes sense